app/dashboard/templates/ens/ens_register.html

```diff
@@ -0,0 +1,23 @@
+{% extends 'ens/ens.html' %}
+{% load static %}
+{% block 'ens_script' %}
+<script src = "{% static 'ens/register.js' %}"></script>
```

Comment:
this file seems to be missing from the PR.  can you pls add it?  same question for `ens/index.js` !

note it should go in the `app/assets/*`directory, not in the `app/static/*` directory!

Reply:
@mbeacom this is the 3rd PR in recent memory where the contributor has put their files in the static dir instead of the assets dir.. where in the documentation do you think we should make a note about the relationship between the assets dir and the static dir?

Reply:
@owocki I say we add it to the contributor documentation and maybe link to django's static docs.  Oddly enough... user changes shouldn't even be recognized in the `static` directory unless they are manually removing the .gitignore entry to commit it. 🤔

Reply:
I will fix this in next commit, thanks.

Reply:
> Oddly enough... user changes shouldn't even be recognized in the static directory unless they are manually removing the .gitignore entry to commit it

i think thats what people are doing.. hehe

Reply:
>  I say we add it to the contributor documentation and maybe link to django's static docs. 

i dont see a real natural place in CONTRIBUTING.md for this.. maybe we add a FAQ section there ?